sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -1,15 +1,13 @@
 # Release History
 
 ## 12.10.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Key Bugs Fixed
-
-### Fixed
-
+- Added support for Immutable Storage with Versioning
+    - Added BlobBaseClient.SetImmutibilityPolicy()
+    - Added BlobBaseClient.DeleteImmutabilityPolicy()
+    - Added BlobBaseClient.SetLegalHold()
+- Added support for listing deleted root blobs with versions to BlobContainerClient.GetBlobs() and .GetBlobsByHierarchy()
+- Added support for OAuth copy sources for synchronous copy operations.
+- Added support for Parquet as an input format in BlockBlobClient.Query().
```

Comment:
Should we also add that it supports service version 2020-10-02, that way people kinda know which service version it will send?

Reply:
Fixed.